### ISSUE TITLE
Need for a test set?

### ISSUE THREAD

**Issue by rahulptel** [alomrani/learning-to-branch#7]
We need a separate test set if the parameters of the model are getting tuned over different runs. 

If we fix the parameters, for example setting the learning rate to 1e-4, before even looking at the dataset then we do not need a test set.

**Comment by rahulptel** [comment]
The hyperparams are fixed without any tuning. Thus, we don't need a test set.
